.github/workflows/frontend.yaml

```diff
@@ -24,6 +24,22 @@ jobs:
       - name: Install dependencies
         run: yarn --frozen-lockfile
 
+      - name: Download graph-federation tar.gz
+        run: wget https://github.com/DiamondLightSource/graph-federation/archive/refs/tags/1.0.0.tar.gz -O graph-federation.tar.gz
```

Comment:
We don't want to pin to a tag here. Otherwise we won't catch issues caused by updating the schema. Might be just worth building the crate and getting the schema that way :thinking:

Reply:
We can just curl the latest tag and use that!

Reply:
I think we want to be building against the current state of the proxy, not the latest tag